### PR TITLE
Improve user guide

### LINKED ISSUE #109: Improve user guide
Based on comments from readers of the user guide, need to amend the explaining and terms used in user guide.

### PR DESCRIPTION
**Pull request by nws321**
This merge aims to address some of the issues mentioned by the other groups for our user guide and command descriptions. It also changes the formatting of the get command to follow closely to sort and help instead. This is because similar to sort and help, get command doesnt require specific parameters like "n/Vincent" but just "name", hence I changed it from "get n/" to "get name". Closes #109.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -153,21 +153,21 @@ Examples:
...
 * All inputted parameters must be valid for the details to be displayed. 
 * The details will be displayed in the order of the inputted parameters.
 * Executing the same parameter multiple times in the same command will still only return one set of that detail.
 
 Examples:
-* `list` followed by `get p/ e/` gets the phone numbers followed by the emails of the people in SocialBook.
-* `list` followed by `get p/ p/` gets only one set of the phone numbers of the people in SocialBook.
-* `find p/HIGH` followed by `get n/ a/` gets the names followed by the addresses of the high priority people.
+* `list` followed by `get phone EMAIL` gets the phone numbers followed by the emails of the people in SocialBook.
+* `list` followed by `get pHone phone` gets only one set of the phone numbers of the people in SocialBook.
```
Comment (arjun2598) [inline docs/UserGuide.md:169]: Is the first "pHone" substring meant to show the case-insensitive nature of the parameters?

**Review by arjun2598 — APPROVED** [review]
LGTM! Thanks for updating the user guide.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -153,21 +153,21 @@ Examples:
...
+* `find p/HIGH` followed by `get nAmE ADDress` gets the names followed by the addresses of the high priority people.
```
Comment (Gra7ityIC3) [inline docs/UserGuide.md]: Shouldn't this be `find pri/HIGH`?

File: docs/UserGuide.md (hunk 1 above)
Comment (Gra7ityIC3) [reply docs/UserGuide.md:169]: Yes, I believe so. The next example also uses mixed-case parameters.

Comment (Gra7ityIC3) [reply docs/UserGuide.md:169]: Yes, I believe so. The next example also uses mixed-case parameters.

File: docs/UserGuide.md (hunk 2 above)
Comment (nws321) [reply docs/UserGuide.md]: u are right thank u for the catch !

**Pull request merged**